packages/loopring_v3/contracts/core/impl/libtransactions/WithdrawTransaction.sol

```diff
@@ -45,19 +45,25 @@ library WithdrawTransaction
         uint    amount;
         uint16  feeTokenID;
         uint    fee;
+        bytes20 dataHash;
```

Comment:
maybe rename `dataHash` to `auxDataHash`, move to the end of the struct, and add comment like:
"// not part of the original Withdrawal data, not signed but supplied by the operator.

Reply:
Can do, but `auxDataHash` may also be a bit confusing because it's not all of the data inside the `WithdrawalAuxiliaryData`, and it's also not really "aux data" because it's just standard withdrawal data that always needs to be there, it's just not necessary for the logic inside the circuit. What about `onchainDataHash`?

Reply:
> Is it possible to only use one single hash for the DA for each tx, and move all DA data to Aux Data?

Maybe too expensive: gas cost for sha256 is `30 + 6 * (size of input in words)`